docs/src/navigation/breadcrumbs/breadcrumbsSample.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Breadcrumbs } from 'react-cm-ui';
+
+const router = { // Router mockup
```

Comment:
The router seems to be a long object, would it be good to include the whole mocked up object in the docs snippet?

Reply:
I think what we have is good.  I'm resolving this thread (we have more discussion up above, which I've left unresolved for visibility ... as afar as I'm concerned this issue is considered resolved).